client/app/bundles/course/reference-timelines/views/DayView/DayView.tsx

```diff
@@ -0,0 +1,113 @@
+import { ComponentRef, useMemo, useRef, useState } from 'react';
+import { Chip, Typography } from '@mui/material';
+import { TimelineData } from 'types/course/referenceTimelines';
+
+import useTranslation from 'lib/hooks/useTranslation';
+
+import DayCalendar from '../../components/DayCalendar';
+import SearchField from '../../components/SearchField';
+import SubmitIndicator from '../../components/SubmitIndicator';
+import TimelinesOverview from '../../components/TimelinesOverview';
+import TimelinesStack from '../../components/TimelinesStack';
+import { selectItems, selectTimelines, useAppSelector } from '../../store';
+import translations from '../../translations';
+
+import ItemsSidebar from './ItemsSidebar';
+
+const DayView = (): JSX.Element => {
+  const { t } = useTranslation();
+
+  const timelines = useAppSelector(selectTimelines);
+  const items = useAppSelector(selectItems);
+
+  const calendarRef = useRef<ComponentRef<typeof DayCalendar>>(null);
+  const contentsRef = useRef<HTMLDivElement>(null);
+
+  const [filterKeyword, setFilterKeyword] = useState('');
+
+  const [hiddenTimelineIds, setHiddenTimelineIds] = useState<
+    Set<TimelineData['id']>
+  >(new Set());
+
+  const visibleTimelines = useMemo(() => {
+    if (!hiddenTimelineIds.size) return timelines;
+
+    return timelines.filter((timeline) => !hiddenTimelineIds.has(timeline.id));
+  }, [hiddenTimelineIds, timelines]);
+
+  const filteredItems = useMemo(() => {
+    if (!filterKeyword) return items;
+
+    return items.filter((item) =>
+      item.title.toLowerCase().includes(filterKeyword.toLowerCase().trim()),
+    );
+  }, [filterKeyword, items]);
+
+  return (
+    <main className="relative flex h-full overflow-hidden">
+      <DayCalendar
+        ref={calendarRef}
+        className="ml-[32rem] border-0 border-l border-solid border-neutral-200"
+        onScroll={(offset): void => {
+          if (contentsRef.current) contentsRef.current.scrollLeft = offset;
+        }}
+      />
+
+      <SubmitIndicator className="absolute right-36 top-0 h-12" />
+
+      <aside className="pointer-events-none absolute top-0 left-0 flex h-full w-full flex-col">
```

Comment:
Because otherwise, the `DayColumn`s header and Today button will not be clickable. Note that this `aside` is on top of `DayCalendar` and covers essentially the entire page.